### ISSUE TITLE
`finance.amount` sometimes returns value with only one decimal place

### ISSUE THREAD

**Issue by Marak** [Marak/faker.js#984]
```
faker.finance.amount()
faker.finance.amount()
faker.finance.amount()
```

```
95.15
669.55
550.8
```

That last one, `550.8` should probably read `550.80` instead.

Would appreciate if anyone could sort this out at:

```
finance.unit.js
  amount(min, max, dec, symbol)
```

**Comment by mastermatt** [comment]
Based on this PR https://github.com/Marak/faker.js/pull/887, `faker.finance.amount()` returns a number instead of a string now. 
So a one digit decimal would be expected in the case of `550.80`.

Note, the JSDoc still says it returns a string. Given the option to pass in a currency symbol, casting the return value to a number seems like the wrong direction.

**Comment by Marak** [comment]
@mastermatt -

Seems like we might have to revert the change for #887 ?

If anyone can get a PR going I will merge.

**Comment by king-11** [comment]
@Marak i would like to work on this issue can i go ahead

**Comment by Marak** [comment]
I always get a little confused when people ask me if they can work on an issue.

The answer is always yes @king-11 

Permission is not required.

**Comment by king-11** [comment]
what i found is that as per ECAMScript specification for Number in any sort of conversion from string to number they use [this](https://tc39.es/ecma262/#sec-tonumeric) specfication where addition is taking place in  StrUnsignedDecimalLiteral:::DecimalDigits.DecimalDigits  part so they don't take into account the zeros. So i guess it won't be possible to have trailing non-significant zeros in the number as per ECMAScript specification.

**Comment by Marak** [comment]
Resolved with #994 🎉
